src/GifPickerReact.tsx

```diff
@@ -23,19 +23,20 @@ export interface GifPickerReactProps {
 	height?: number | string;
 	categoryHeight?: number | string;
 	theme?: Theme;
+	pickerContext?: [PickerContextType, Dispatch<SetStateAction<PickerContextType>>];
```

Comment:
If the only reason for this PR is adding the ability to change the initial search it's not a proper solution. I am strongly against exposing internal context. I would suggest adding `initialSearchTerm` property here. That would fit your use cases and is probably is the most intuitive solution

Reply:
I will work on that